src/states/Cutscene.js

```diff
@@ -1,4 +1,20 @@
-var cutscene = function () {};
+var Cutscene = function () {};
```

Comment:
We may need to rethink our naming convention that we discussed. Making state names capital makes sense, but we had said variables and functions are camelCase.